prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -362,13 +362,6 @@ class Hl7Serializer(
             errors.add(msg)
         }
 
-        // Check for required fields now that we are done processing all the fields
-        schema.elements.forEach { element ->
-            if (!element.isOptional && mappedRows[element.name]!!.isBlank()) {
-                errors.add("The Value for ${element.name} for field ${element.hl7Field} is required")
-            }
-        }
-
```

Comment:
I removed this because it is a duplicate of what is in the processValue function and hence we get two of the same warning/errors.

> I admit I don't understand this code - but this check looks important -- just doublechecking that its really ok to remove it?